.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,59 @@
+name: Integration-Test
+on:
+  workflow_call:
+env:
+  GO_VERSION: "~1.18"
+  CONTROLLER_TOOLS_VERSION: "v0.9.2"
+  ENVTEST_K8S_VERSION: "1.24.2"
+  SCHEDULER_COMPATIBLE_K8S_VERSION: "v0.24.3"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  run-integration-test:
+    name: Run Integration Tests
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          cache: true
+          cache-dependency-path: '**/go.sum'
+
+      - name: Download manifests
+        uses: actions/download-artifact@v3
+        with:
+          path: ~/download/artifacts
+
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1.3.0
+  
+      - name: Install cert-manager
+        run: |
+          kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.10.0/cert-manager.yaml
```

Comment:
Please also move the version number here into an env variable to the top of the file.